README.md

```diff
@@ -1,76 +1,217 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+<!-- PROJECT DESCRIPTION -->
 
-## Scripts
+<!-- this is my new blog I created with HTML, CSS -->
 
-The following scripts are available in the project:
+# Leaderboard
+This project is Leaderboard microvers which will be used for learning in the microverse. No complex codind is required for this exercise. The goal is to master all of the tools and best practices learned in previous steps. They will be used in all Microverse projects and most likely in future jobs as well, so it is important to know!.
+1- HTML
+2- css
+3- linters
+4- javascript
+5- Webpack
 
-- `npm start`: Starts the development server.
-- `npm run build`: Builds the project for production.
-- `npm run lint`: Runs ESLint and Stylelint to check for linting issues.
-- `npm test`: Runs the project's tests (if any).
-- `npm run deploy`: Deploys the project to GitHub Pages.
+## 🛠 Built With <a name="built-with"></a>
 
-## Configuration
+### Tech Stack <a name="tech-stack"></a>
 
-The project includes default configurations for ESLint and Stylelint, located in the root directory:
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">css</a></li>
+  </ul>
+</details>
 
-- `.eslintrc.json`: ESLint configuration file with recommended settings.
-- `.stylelintrc.json`: Stylelint configuration file with recommended settings.
+<!-- Features -->
 
-You can modify these configuration files according to your project's requirements.
+### Key Features <a name="key-features"></a>
+- **Demo of Linters**
+- **Demo of LeaderBoard**
 
-## Pre-commit Hooks
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-The project is configured to run the linters automatically before committing changes, ensuring that code quality and style standards are maintained. If any issues are found, the commit will be blocked until the issues are resolved.
+# LIVE DEMO 
 
-## Deploying to GitHub Pages
+- [] coming soon
 
-To deploy your project to GitHub Pages, use the following script:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-```bash
-npm run deploy
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1- web browser
+2- git-syn
+3- code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+  cd my-folder
+  git clone https://github.com/axcell-0/Leaderboard.git
+
+### Install
+
+Install this project with:
+
+
+Run this command:
+
+...sh
+  cd my-project
+  npm install
+  ...
+
+### Usage
+
+To run the project, execute the following command:
+-Run
+
+  npm run build
+
+-then
+
+  npm run dev
```

Comment:
- [ ] To use your project users need to run `npm start` not `npm run dev`. Because you currently have only a start script which is using webpack-serve to display the project on the user browser, there are no scripts for `dev` which would enable anyone display your project on their browser. I kindly suggest you make this correction to this part of your readme file. 👍